### PR TITLE
feat(workerpool): update to version 6.4

### PR DESCRIPTION
**Pull request by Michsior14**
Updates the workerpool types to version 6.4.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [here](https://github.com/josdejong/workerpool/compare/v6.1.0...v6.4.0)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## workerpool ([<kbd>unpkg</kbd>](https://unpkg.com/browse/workerpool@latest/))
was missing the following properties:
1. Transfer
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 8702c2e8014510a8c563c2dd090a23af40034068
</p>

**Comment by forivall** [comment]
I notice that you checked the "Provide a URL to documentation or source code which provides context for the suggested changes:", but I dont see a URL? PR looks good, but I'd like to see their docs!

**Comment by Michsior14** [comment]
> I notice that you checked the "Provide a URL to documentation or source code which provides context for the suggested changes:", but I dont see a URL? PR looks good, but I'd like to see their docs!

Sorry, formatting issue, fixed ;)

**Review by forivall — CHANGES_REQUESTED** [review]
from the docs, looks like the type signatures arent quite right.

File: types/workerpool/index.d.ts [hunk 1]
```diff
@@ -57,6 +59,20 @@ export interface WorkerPool {
...
+ */
+export class Transfer {
+    message: any;
+    transfer: any[];
+
+    /**
+     * @param message The object to deliver to the main thread.
+     * @param transfer An array of transferable Objects to transfer ownership of.
+     */
+    constructor(message: any, transfer: any[]);
```
Comment (forivall) [inline types/workerpool/index.d.ts:73]: it requires the dom typing, but `transfer` should be `Transferable[]`
```suggestion
    constructor(message: any, transfer: Transferable[]);
```

**Review by forivall — APPROVED** [review]
LGTM! thanks

**Comment by Michsior14** [comment]
Ready to merge

**Pull request merged**